FrameworklessServerTests/UsersServiceTests.cs

```diff
@@ -111,5 +105,27 @@ public void GivenUserNameShouldReturnNameFromList()
             Assert.Equal(expected.Name, result.Name);
             ResetList();
         }
+
+        [Fact]
+        public void GivenUserNameShouldUpdateToNewName()
+        {
+            _usersService.Add(new User("Bob"));
+            var expected = new List<User> {new User("Cindy"), new User("Bob")};
+            Assert.Equal(expected[1].Name, _usersService.GetAllUsers()[1].Name);
+
+            _usersService.UpdateUser("Bob", "Mary");
+            
+             expected = new List<User> {new User("Cindy"), new User("Mary")};
+            Assert.Equal(expected[1].Name, _usersService.GetAllUsers()[1].Name);
+            ResetList();
```

Comment:
This is a bad practice. A better way would be to use the constructor to do the setup.